bin/build

```diff
@@ -2,26 +2,47 @@
 
 set -e
 
-. bin/build_utils
+# Use Docker BuildKit to efficiently build the targets.
+# With BuildKit, only relevant layers for the target are built.
+export DOCKER_BUILDKIT=1
+DEBUG=false
 
-FULL_VERSION_TAG="$(full_version_tag)"
+if [ "$1" == "--debug" ]; then
```

Comment:
I really prefer that we handle this in a way that it is easy to add more params in the future. Please see the way that we handled this in `./bin/test_integration`. Do you think it's not the correct way to do it?